RELEASING.md

```diff
@@ -0,0 +1,186 @@
+versioning](https://semver.org/) (semver):
+
+* `v1.x.x` - MAJOR version, increments on breaking changes.
+* `v1.2.x` - MINOR version, increments on backwards-compatible new features.
+* `v1.2.3` - PATCH version, increments on backwards-compatible bug fixes.
+
+Examples of potential breaking changes:
+
+* The Engine drops support for .yaml config files.
+* A [CFT module](https://cloud.google.com/foundation-toolkit) module MAJOR version bump
+  (i.e. [terraform-google-kubernetes-engine](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine)).
+* A Terraform provider MAJOR version bump.
+* Terraform itself has a major version update and drops a feature we relied on, causing us to heavily modify the templates.
+
+### Release Tags
+
+We use the release tags `vX.X.X` for binaries, `templates-vX.X.X` for templates, and `policies-vX.X.X` for policies.
+All templates are versioned together as one unit, and so are all policies.
+
+Examples:
+
+* [`v0.1.0`](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/tag/v0.1.0)
+  for a binaries release
+* [`templates-v0.1.0`](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/tag/templates-v0.1.0)
+  for a templates release
+* [`policies-v0.1.0`](https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/releases/tag/policies-v0.1.0)
+  for a policies release.
+
+### The 0 Version
+
+The only exception to the above is the `0.x.x` version. As described in semver, this version is for initial development;
+anything may change at any time, and the public API should not be considered stable. During this time, we will bump
+the 0.MINOR version for breaking changes, and the 0.x.PATCH version for all other changes.
+
+### Release Branching
+
+For now, we do not preemptively create release branches. Release tags are made on master by Repo Owners, and we
+primarily work on the latest version of the binaries, templates and policies.
+
+Release branches may be cut from old releases as necessary, but we would generally encourage using or migrating to the
+latest versions of everything.
+
+### Frequency
+
+We strive to make a release at least once a month, containing accumulated changes since the last release. We may make
+out-of-band releases any time as needed. We will bump the MAJOR/MINOR/PATCH versions as needed, without guarantees on
+when the MINOR or MAJOR versions will be bumped.
+
+### Caveat about versions
+
+Solutions docs and guides should pin the versions of the binaries, templates and policies to ensure that users will see
+consistent binary behaviour and output files.
+
+However, as cloud APIs and other infrastructure change over time, in practice solutions may stop working even with the
+same versions as before. We cannot guarantee indefinite compatibility, and encourage solutions and guide writers to
+periodically check for correctness and consider updating to use the latest versions.
+
+## Automated releases
```

Comment:
Should these go in a dev doc? I think this doc should only be for users

Reply:
I personally don't think so, I see this doc as the one place for release-related information. Also in the past where I've encountered RELEASING.md docs, they have instructions on how to make a release.